docs/ml-features.md

```diff
@@ -211,6 +211,87 @@ for feature in result.select("result").take(3):
 </div>
 </div>
 
+## CountVectorizer
+
+As a transformer, `CountVectorizerModel` converts a collection of text documents to vectors of token counts.
```

Comment:
Shall we start with `CountVectorizer` but not `CountVectorizerModel`? I guess most users would use `CountVectorizer` to build the vocabulary.